src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/communitypage/SalesPitchDialog.java

```diff
@@ -0,0 +1,39 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.communitypage;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class SalesPitchDialog extends WikiBasePageObject {
+
+  private By salesPitchDialog = By.cssSelector("#CommunityPageBenefitsModal");
+
+  @FindBy(css = "#CommunityPageBenefitsModal .community-page-entry-point-button")
+  private WebElement entryPointButton;
+
+  public SalesPitchDialog() { super(); }
```

Comment:
IIRC our coding conventions for Java would requires these to be split over multiple lines.